x/superfluid/keeper/internal/events/emit_test.go

```diff
@@ -0,0 +1,339 @@
+package events_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v10/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v10/x/superfluid/keeper/internal/events"
+	"github.com/osmosis-labs/osmosis/v10/x/superfluid/types"
+)
+
+type SuperfluidEventsTestSuite struct {
```

Comment:
Just wondering, what was the motivation behind create a new test suite? Can't we just use the `KeeperTestSuite`?

cc @p0mvn